packages/plugin/src/Fields/Implementations/Pro/PasswordField.php

```diff
@@ -26,6 +27,9 @@
 )]
 class PasswordField extends TextField implements NoStorageInterface, ExtraFieldInterface, RememberPostedValueInterface
 {
+    #[Input\Hidden]
```

Comment:
@gustavs-gutmanis This is hiding the Encrypt field data label/checkbox. Because Password extends Text, Encrypt field data was still showing as 1) its within Pro edition and 2) inheritances EncryptionInterface from Text (which cannot be removed).

Reply:
TBH, I'm not 100% happy with this as a solution. Setting encrypted to false alone doesn't hide it. Unless you know why, a dev would be very confused. What do you think about making Password not extend Text?